tests/app/strings.js

```diff
@@ -11,6 +11,8 @@ describe('strings', function() {
     expect(stringsAnswers.reduceString('aaxxxaabbbb', 2)).to.eql('aaxxaabb');
   });
 
+
+  
```

Comment:
merge/pull request is also a good place to check if you have changed only files you were working on and not anything else (like supporting files in test folder here)